### PR TITLE
Add an arabic adventure

### PR DESCRIPTION
**Pull request by AliAlsayed**
Wanted to add this for practice. Thank you

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: arabic-adventure/AlsalamAlaikom.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Al salam alikom
```
Comment (casey-collab) [inline arabic-adventure/AlsalamAlaikom.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
This looks like a test pull request to me, which is fine, but I'm going to close it since I only want to merge contributions to the story. I don't speak Arabic, though, so if I've misunderstood, please re-open the pull request and let me know.

**Pull request closed without merging**